internal/declarative/v2/ssa_test.go

```diff
@@ -16,15 +16,17 @@ import (
 func TestConcurrentSSA(t *testing.T) {
 	t.Parallel()
 
-	pod := &unstructured.Unstructured{Object: map[string]any{
-		"kind":       "Pod",
-		"apiVersion": "v1",
-		"metadata": map[string]interface{}{
-			"name":      "valid",
-			"namespace": "some-namespace",
+	pod := &unstructured.Unstructured{
+		Object: map[string]any{
+			"kind":       "Pod",
+			"apiVersion": "v1",
+			"metadata": map[string]interface{}{
+				"name":      "valid",
+				"namespace": "some-namespace",
+			},
 		},
-	}}
-	structuredInfo := &resource.Info{Object: pod}
+	}
+	// structuredInfo := &resource.Info{Object: pod}
```

Comment:
same here, if it's not in use, just remove it.

Reply:
https://github.com/orgs/kyma-project/projects/30/views/13?pane=issue&itemId=53973979